apps/web/src/components/layouts/AppLayout/index.tsx

```diff
@@ -19,17 +17,14 @@ export default function AppLayout({
   sectionLinks,
 }: AppLayoutProps) {
   return (
-    <AppLayoutProvider
```

Comment:
Did you remove this provider?

Reply:
Yes! We no longer need to store the `contentHeight`.